src/Presenter/LinkBlockPresenter.php

```diff
@@ -215,4 +220,28 @@ private function makeCustomLinks($customContent)
 
         return $customLinks;
     }
+
+    /**
+     * @param array $categoryIds
+     *
+     * @return array
+     */
+    private function makeCategoryLinks($categoryIds)
+    {
+        $categoryLinks = array();
+        foreach ($categoryIds as $categoryId) {
+            if (false !== $categoryId) {
+                $meta = \Meta::getCategoryMetas($categoryId, (int) $this->language->id, null, null);
+                $categoryLinks[] = array(
+                    'id' => 'link-category-' . $categoryId,
+                    'class' => 'category-link',
+                    'title' => $meta['name'],
+                    'description' => strip_tags($meta['description']),
+                    'url' => $this->link->getCategoryLink((int)$categoryId),
+                );
+            }
```

Comment:
```suggestion
            if (false === $categoryId) {
                continue;
            }

            $meta = \Meta::getCategoryMetas($categoryId, (int) $this->language->id, null, null);
            $categoryLinks[] = array(
                'id' => 'link-category-' . $categoryId,
                'class' => 'category-link',
                'title' => $meta['name'],
                'description' => strip_tags($meta['description']),
                'url' => $this->link->getCategoryLink((int)$categoryId),
            );
```

Reply:
Agreed, this is more readable.

All the other methods use the same format, changed all of them (0193854)

Not 100% why the check even is there, I cant see how the id would be not defined there but its always good to be defensive.

I don't see any test cases for this module.. are they in core repo or are the modules not tested?